etc/inc/dyndns.class

```diff
@@ -1288,6 +1299,33 @@
 						$this->_debug($body);
 					}
 					break;
+				case 'dnsmadeeasy':
+					if (preg_match('/success/i', $data)) {
```

Comment:
All these preg_match() calls are an overhead. IMO all these if/else blocks should be replace by a switch case block